device/rsvp/rsvpIngressP2PLsp.yaml

```diff
@@ -0,0 +1,91 @@
+components:
+  schemas:
+    Rsvp.LspInterface.IngressP2PLSP:
+      description: >-
+        Configuration for one RSVP Ingress point-to-point LSP.
+      type: object
+      required: [remote_address]
+      properties:
+        remote_address:
+          description: >-
+            IPv4 address of the remote endpoint of the LSP.
+          type: string
+          format: ipv4
+        tunnel_id:
+          description: >-
+            The Tunnel ID of the RSVP LSP. Carried in the SESSION object in Path Messages.
+          type: integer
+          default: 1
+          minimum: 1
+          maximum: 65535
+        lsp_id:
+          description: >-
+            The LSP ID of the RSVP LSP. Carried in the SENDER_TEMPLATE object in Path Messages.
+          type: integer
+          default: 1
+          minimum: 1
+          maximum: 65535
+        refresh_interval:
+          description: >-
+             The refresh interval in seconds after which refresh Path Messages are periodically sent. 
+             The actual refresh interval is jittered by upto 50%.
+             There is no specification specified maximum value. For clarity, setting the maximum to 1 hour. 
+          type: integer
+          default: 30
+          minimum: 1
+          maximum: 3600
+        timeout_multiplier:
+          description: >-
+             The number of missed Path refreshes after which a recieving node should consider the LSP state to have timed out.
+             There is no specification specified maximum value. Setting the maximum allowed value to 10 .
+          type: integer 
+          default: 3
+          minimum: 1
+          maximum: 10
+        backup_lsp_id:
+          description: >-
+             The LSP id that will be used when creating a Make-Before-Break LSP when the active LSP is using lsp_id.
+             If the active LSP on which Make-Before-Break is being done is using the backup_lsp_id, the new LSP created will toggle to 
+             use the lsp_id instead.
+          type: integer 
+          default: 30000
+          minimum: 1
+          maximum: 65535
+        lsp_switchover_delay:
+          description: >-
+            The amount of delay in seconds that an implementation should wait for before switching traffic to the new LSP created after 
+            a Make-Before-Break is done on an LSP. The default value is 0 which means to switch immediately.
+            There is no specification specified maximum value. Setting maximum allowed value to 1 minute. 
+          type: integer
+          default: 0
+          minimum: 0
+          maximum: 60
+        reevaluation_request_interval:
+          description: >-
+            If value is set to '0' , don't send periodic Path Re-optimization requests.
+            If value is set to a non-zero value, send Path Re-optimization requests at interval of specified interval (in seconds).
```

Comment:
If value is set to a non-zero value, send Path Re-optimization requests at interval of specified non-zero value (in seconds).

Reply:
Done.